app/questionnaire/navigation.py

```diff
@@ -44,20 +44,22 @@ def build_navigation(self, current_group_id, current_group_instance):
 
             skip_group = self._should_skip_group(current_group_instance, group)
             last_block_in_current_group = group['blocks'][-1]
```

Comment:
I know this would be from the original PR but think as its a for loop for 'group' we don't need to define _current_group, additionally we are putting things like group['blocks'][-1] in SchemaHelper (it might already be in there).